plugins/source/aws/resources/services/organizations/organizational_units.go

```diff
@@ -0,0 +1,96 @@
+package organizations
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/organizations"
+	"github.com/aws/aws-sdk-go-v2/service/organizations/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client/services"
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+)
+
+func OrganizationalUnits() *schema.Table {
+	return &schema.Table{
+		Name:                "aws_organizations_organizational_units",
+		Description:         `https://docs.aws.amazon.com/organizations/latest/APIReference/API_OrganizationalUnit.html`,
+		Resolver:            fetchOUs,
+		PreResourceResolver: getOU,
+		Transform: transformers.TransformWithStruct(
+			&types.OrganizationalUnit{},
+			transformers.WithPrimaryKeys("Arn"),
+		),
+		Multiplex: client.ServiceAccountRegionMultiplexer("organizations"),
+		Columns:   []schema.Column{client.DefaultAccountIDColumn(true)},
+	}
+}
+
+func fetchOUs(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- any) error {
+	c := meta.(*client.Client)
+	svc := c.Services().Organizations
+	var input organizations.ListRootsInput
+	paginator := organizations.NewListRootsPaginator(svc, &input)
+	var roots []types.Root
+	for paginator.HasMorePages() {
+		page, err := paginator.NextPage(ctx)
+		if err != nil {
+			return err
+		}
+		roots = append(roots, page.Roots...)
+	}
+
+	for _, root := range roots {
+		err := getOUs(ctx, svc, res, *root.Id)
+		if err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func getOUs(ctx context.Context, accountsApi services.OrganizationsClient, res chan<- any, parentID string) error {
+	q := []string{parentID}
+	var ou string
+	seenOUs := map[string]struct{}{}
+
+	for len(q) > 0 {
```

Comment:
I thought this code looked familiar / quite like my own style, then I remembered I wrote it a while back for the AWS client. LGTM :)

Reply:
100% I copied that straight from your work in the AWS Client...